api/invite.go

```diff
@@ -94,9 +94,9 @@ func (c *Client) CreateInvite(
 // JoinedInvite is returned after joining an invite.
 type JoinedInvite struct {
 	Code      string          `json:"code"`
-	NewMember bool            `json:"new_member"`
+	Channel   discord.Channel `json:"channel"`
```

Comment:
Add EOL comment `// id, name, type only`.

Reply:
```suggestion
	// Channel is the channel that was joined.
	// Only ID, Name and Type will be set.
	Channel   discord.Channel `json:"channel"`
```
This should probably be a full field comment, i.e. above `Channel`, so it shows up on tooltips etc. of development environments that offer views of viewing the docs other than viewing at the source code.